x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_nav.tsx

```diff
@@ -171,8 +228,33 @@ export const EngineNav: React.FC = () => {
           to={getAppSearchUrl(engineRoute + ENGINE_RELEVANCE_TUNING_PATH)}
           data-test-subj="EngineRelevanceTuningLink"
         >
-          {RELEVANCE_TUNING_TITLE}
-          {/* TODO: invalid boosts error icon */}
+          <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none">
+            <EuiFlexItem>{RELEVANCE_TUNING_TITLE}</EuiFlexItem>
+            <EuiFlexItem className="appSearchNavIcons">
```

Comment:
I debated pulling this out into a separate component called EngineNavAlerts or something, but eventually decided against it for now - it felt like a pre-optimization and tests felt easier to write and more clear w/o it. We can definitely do so if we start adding more engine nav icons in the future though.